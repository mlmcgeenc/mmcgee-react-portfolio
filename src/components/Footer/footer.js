import React from 'react';
import 'bootstrap';
import '../../Assets/css/custom.css';
import CodepenLogo from '../../Assets/images/codepen-brands.svg';
import GithubLogo from '../../Assets/images/github-brands.svg';
import LeetcodeLogo from '../../Assets/images/leetcode.svg';
import LinkedinLogo from '../../Assets/images/linkedin-brands.svg';


const Footer = () => {
	return (
		<div className='justify-content-center fixed-bottom footer'>
			<a className='me-4' href='https://github.com/mlmcgeenc'>
				<img className='footer-logo' src={GithubLogo} alt='github logo'></img>
			</a>
			<a className='mx-4' href='https://www.linkedin.com/in/mattmcgeenc/'>
				<img className='footer-logo' src={LinkedinLogo} alt='linked in logo'></img>
			</a>
			<a className='mx-4' href='https://codepen.io/mlmcgeenc'>
				<img className='footer-logo' src={CodepenLogo} alt='codepen logo'></img>
			</a>
			<a className='ms-4' href='https://leetcode.com/mlmcgeenc/'>
				<img className='footer-logo' src={LeetcodeLogo} alt='leetcode logo'></img>
			</a>
		</div>
	);
};

export default Footer;
