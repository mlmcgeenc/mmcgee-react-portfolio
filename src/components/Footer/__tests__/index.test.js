import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Footer from '../footer';

afterEach(cleanup);

describe('Footer component', () => {
	it('renders', () => {
		render(<Footer />);
	});

	it('matches snapshot DOM node structure', () => {
		const { asFragment } = render(<Footer />);
		expect(asFragment()).toMatchSnapshot();
	});
});
