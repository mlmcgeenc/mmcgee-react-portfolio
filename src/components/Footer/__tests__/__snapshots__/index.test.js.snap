// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    Footer
  </div>
</DocumentFragment>
`;
